datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/tls.yaml

```diff
@@ -0,0 +1,50 @@
+- name: tls_verify
+  value:
+    example: true
+    type: boolean
+  description: |
+    Instructs the check to validate the TLS certificate of services.
+- name: tls_ca_cert
+  value:
+    example: <CA_CERT_PATH>
+    type: string
+  description: |
+    The path to a file of concatenated CA certificates in PEM format or a directory
+    containing several CA certificates in PEM format. If a directory, the directory
+    must have been processed using the c_rehash utility supplied with OpenSSL. See:
+    https://www.openssl.org/docs/manmaster/man3/SSL_CTX_load_verify_locations.html
+
+    Setting this implicitly sets `tls_verify` to true.
```

Comment:
If it is possible to implicitly set `tls_verify` to true, wouldn't it make the actual `tls_verify` option less relevant?

Reply:
Not necessarily, `tls_verify` can be enabled/disabled on its own without having to change the other parameters